integrationtests/src/test/java/io/kroxylicious/proxy/BaseIT.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.proxy;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import org.apache.kafka.clients.admin.Admin;
+import org.apache.kafka.clients.admin.CreateTopicsResult;
+import org.apache.kafka.clients.admin.DeleteTopicsResult;
+import org.apache.kafka.clients.admin.NewTopic;
+import org.apache.kafka.clients.consumer.Consumer;
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.common.TopicCollection;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import io.kroxylicious.test.tester.KroxyliciousTester;
+import io.kroxylicious.testing.kafka.junit5ext.KafkaClusterExtension;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@ExtendWith(KafkaClusterExtension.class)
+public abstract class BaseIT {
+
+    protected static CreateTopicsResult createTopics(Admin admin, List<NewTopic> topics) {
+        try {
+            var created = admin.createTopics(topics);
```

Comment:
+1 to this approach.

It's a pity there's no way we can roll the default future timeout into CloseableAdmin and friends somehow.   I don't see a reasonable way to do that.  I considered dynamic proxies, but .. urgh.  Maybe someone has a clever unobtrusive way.

Reply:
No but if we stop having static methods we can do something funky like:
```java
   private CreateTopicsResult createTopics(Admin admin, List<NewTopic> topics) throws Exception {
        CreateTopicsResult created = executeRpc(() -> admin.createTopics(topics));
        assertEquals(topics.size(), created.values().size());
        return created;
    }

    @NotNull
    private <R> R executeRpc(Supplier<KafkaFuture<R>> futureSupplier) throws InterruptedException, ExecutionException, TimeoutException {
        try {
            return futureSupplier.get().get(FUTURE_TIMEOUT_SECONDS, TimeUnit.SECONDS);
        }
        catch (ExecutionException e) {
            throw new RuntimeException(e.getCause());
        }
        catch (InterruptedException | TimeoutException e) {
            throw new RuntimeException(e);
        }
    }
```

Reply:
You are right. I'm hoping that the idea of moving a lot of this stuff to the tester pans out and we contain it better even if still doing it manually.